packages/mdc-fab/_mixins.scss

```diff
@@ -122,12 +127,28 @@ $mdc-fab-icon-enter-duration_: 180ms;
   height: 40px;
 }
 
+@mixin mdc-fab--extended_ {
+  width: auto;
+  max-width: 100%;
+  height: $mdc-fab-extended-height_;
+  padding: 0 20px 0 16px;
+  border-radius: $mdc-fab-extended-height_ / 2;
+  white-space: nowrap;
```

Comment:
Would this style make more sense specifically on the label element? (Tangentially, flex-wrap is already nowrap by default)